src/Reflector.tsx

```diff
@@ -16,7 +16,7 @@ declare global {
   }
 }
 
-type Props = ReflectorOptions & { children: React.ReactElement<any> }
+type Props = ReflectorOptions & Omit<JSX.IntrinsicElements['mesh'], 'args'> & { children: React.ReactElement<any> }
```

Comment:
The THREE class `Reflector` extends `Mesh` so the story failed, although I find the TS for R3F confusing (becuase im no master) so this might be wrong?